transform/v2/tsv-to-jsonl.py

```diff
@@ -0,0 +1,121 @@
+import fileinput
```

Comment:
Have you written the module/class or its copy-paste solution?

Reply:
It's copied and then modified to match the data we get.